ras_backstage/resources/survey/get_survey_by_short_name.py

```diff
@@ -0,0 +1,27 @@
+import logging
+
+from flask import jsonify, make_response
+from flask_restplus import Resource
+from structlog import wrap_logger
+
+from ras_backstage import survey_api
+from ras_backstage.controllers import collection_exercise_controller, survey_controller
+
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+@survey_api.route('/shortname/<string:short_name>')
```

Comment:
It's worth bearing in mind if we need to get this information by any other query i.e. survey id we could revisit this API and make it based on query params